Source/Scene/Cesium3DTileBatchTableResources.js

```diff
@@ -2,14 +2,19 @@
 define([
         '../Core/arrayFill',
         '../Core/Cartesian2',
+        '../Core/Cartesian3',
```

Comment:
Should we rename:
* `Cesium3DTileBatchTableResources` -> `Cesium3DTileBatchTable`

I think I added "Resources" because of the batch table texture, but perhaps this rename would be more concise and mirror `Cesium3DFeatureTable`.